src/domain/community/community.repository.interface.ts

```diff
@@ -14,6 +15,11 @@ export interface ICommunityRepository {
     offset?: number;
   }): Promise<Page<Campaign>>;
 
+  getCampaignActivities(args: {
+    resourceId: string;
+    holder?: `0x${string}`;
```

Comment:
```
  limit?: number;
  offset?: number;
```

This is missing in the function interface, right?

Reply:
It was! Added in 4d67562054be2d64dd9b8a6648fede25cf685324.